README.md

```diff
@@ -885,6 +891,51 @@ If these failures occur, add the warning to the list of permitted warnings found
 in [`AzulTestCase`](test/azul_test_case.py) and commit the modifications. 
 
 
+## Installing Python 3.8.3 on macOS BigSur
+
+[pyenv Big Sur GitHub issue](https://github.com/pyenv/pyenv/issues/1740)
+
+macOS Big Sur users may encounter a `build failed` error when installing Python through pyenv. 
+To remedy this there is a patch file which runs the installer compatible with 
+macOS Big Sur (11.x).
+
+First is to ensure that all requirements for the Python build environment 
+from the pyenv [wiki](https://github.com/pyenv/pyenv/wiki#suggested-build-environment) 
+are met. We extend their suggestion and also install bzip2.
+
+```
+brew install openssl readline sqlite3 xz zlib bzip2
+```
+
+Follow any additional steps that `brew` requires for installation, these 
+should include modifying path variables `LDFLAGS` and `CPPFLAGS`
+The commands from the brew output to modify the aforementioned path variables 
+can be placed in the `~/.bash_profile` to make the change persistent
+
+Install python 3.8.3 using pyenv by running:
+
+```
+pyenv install --patch 3.8.3 < <(curl -sSL https://github.com/python/cpython/commit/8ea6353.patch)
+```
+
+
+## Failure to locate wheels on macOS Big Sur
```

Comment:
People may think they are done when the previous subsection ends. This should be the third step in a single section.